knowledge_base/Groups.md

```diff
@@ -7,6 +8,16 @@
   * [Update/delete Group](#update-delete-group)
 - [Change Log](#change-log)
 
+# Introduction to Groups
+
+`Groups` are used by community admins to handle permissioning across the forum, using associated `Membership` objects that link `Addresses` to Groups. 
+
+Forum actions are always topic-scoped; whenever a user creates a new thread, reacts to a comment, or participates in a poll, they are operating within the scope of a topic. Groups allow admins to "gate" user actions, within a given topic-scope, based on a set of Requirements.
```

Comment:
Can we add an "e.g. creating a thread or a comment" to "user actions" so we're clear about what kinds of actions we mean?

Reply:
Edited for clarity!